content/livesync/subscriptions.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Subscriptions
+product: Livesync
+---
+
+After setting up the necessary registrations, you can subscribe to your data model. This subscription enables real-time updates whenever there's a change in the data:
+
+```
+await model.pause();
+// model paused: new events will not be processed and subscription callbacks will no longer be invoked
+await model.resume();
+// processing of events has resumed and new changes will be made available to subscribers
+```
+
+By default, subscriptions operate on an optimistic model, meaning the callback is triggered by any change in the optimistic state or when confirmed changes occur. 
```

Comment:
"operate on" sounds plays some active roll, but subscribing to updates is passive, so this sounds odd to me